config/prometheus/crd.yaml

```diff
@@ -0,0 +1,266 @@
+---
```

Comment:
Like @rainest commented above:

> We shouldn't include the PodMonitor CRD ourselves. That should come from the Prometheus operator. We should only provide the PodMonitor/ServiceMonitor resources alone.

Please delete the CRD from the kustomization.

(same comment as for `deploy/`)

Reply:
changed b7c4fbb5d58fd7878a29a0251f69d11ac59c3a62 accordingly